entropylab/results_backend/sqlalchemy/alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
```suggestion
This alembic environment is responsible for migrating the database for the results metadata Entropy save for each run
```

Reply:
Make the readme more informative

Reply:
I will expand this README to explain how to create new migrations etc tomorrow 🤞